packages/@aws-cdk/aws-events/README.md

```diff
@@ -89,6 +89,7 @@ import { Rule, Schedule } from '@aws-cdk/aws-events';
 import { EcsTask } from '@aws-cdk/aws-events-targets';
 ...
 
+// use `LATEST` as platformVersion by default
```

Comment:
it's not really clear what part of the next line this comment applies to. Let's add more detail here and get more specific about where the platformVersion default comes from.

Reply:
fixes: de0cb76